moveit_ros/move_group/src/move_group_context.cpp

```diff
@@ -40,14 +40,27 @@
 #include <moveit/plan_execution/plan_execution.h>
 #include <moveit/plan_execution/plan_with_sensing.h>
 
-move_group::MoveGroupContext::MoveGroupContext(
-    const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor, bool allow_trajectory_execution,
-    bool debug)
+namespace move_group
+{
+ros::NodeHandle MoveGroupContext::getConfigNodeHandle()
+{
+  ros::NodeHandle nh = ros::NodeHandle("~");
```

Comment:
```suggestion
  ros::NodeHandle nh("~");
```

Reply:
nit: I think it would be more elegant to initialize `default_pipeline` with "~" and query the parameter afterwards, so that `nh` is only initialized once with the resulting namespace.

Reply:
Sorry, I don't yet understand your suggestion. `default_pipeline` is a ROS parameter denoting the desired pipeline, e.g. `ompl`. How would initializing it with `"~"` help? Maybe just draft your suggested change?